test/cppsuite/test_harness/workload_tracking.h

```diff
@@ -30,39 +30,79 @@
 #define WORKLOAD_TRACKING_H
 
 /*
- * Default schema for tracking table key_format : Collection name / Key value_format : Operation
- * type / Value / Timestamp
+ * Default schema for tracking operations on collections (key_format: Collection name / Key /
+ * Timestamp, value_format: Operation type / Value)
  */
-#define DEFAULT_TRACKING_KEY_FORMAT WT_UNCHECKED_STRING(Si)
-#define DEFAULT_TRACKING_VALUE_FORMAT WT_UNCHECKED_STRING(iSi)
-#define DEFAULT_TRACKING_TABLE_SCHEMA \
-    "key_format=" DEFAULT_TRACKING_KEY_FORMAT ",value_format=" DEFAULT_TRACKING_VALUE_FORMAT
+#define OPERATION_TRACKING_KEY_FORMAT WT_UNCHECKED_STRING(Sii)
+#define OPERATION_TRACKING_VALUE_FORMAT WT_UNCHECKED_STRING(iS)
+#define DEFAULT_OPERATION_TRACKING \
+    "key_format=" OPERATION_TRACKING_KEY_FORMAT ",value_format=" OPERATION_TRACKING_VALUE_FORMAT
+
+/*
+ * Default schema for tracking schema operations on collections (key_format: Collection name /
+ * Timestamp, value_format: Operation type)
+ */
+#define SCHEMA_TRACKING_KEY_FORMAT WT_UNCHECKED_STRING(Si)
+#define SCHEMA_TRACKING_VALUE_FORMAT WT_UNCHECKED_STRING(i)
+#define DEFAULT_SCHEMA_TRACKING \
+    "key_format=" SCHEMA_TRACKING_KEY_FORMAT ",value_format=" SCHEMA_TRACKING_VALUE_FORMAT
 
 namespace test_harness {
 /* Tracking operations. */
-enum class tracking_operation { CREATE, INSERT };
+enum class tracking_operation { CREATE, DELETE_COLLECTION, DELETE_KEY, INSERT };
 /* Class used to track operations performed on collections */
-class workload_tracking {
+class workload_tracking : public component {
 
     public:
-    workload_tracking(const std::string &collection_name)
-        : _collection_name(collection_name), _timestamp(0U)
+    workload_tracking(const std::string &collection_operations_name = TABLE_OPERATION_TRACKING,
+      const std::string &collection_schema_name = TABLE_SCHEMA_TRACKING,
```

Comment:
```
const std::string operation_table_name,
const std::string schema_table_name,
collections_operations_format->operation_table_format.
collections_schema_format -> schema_table_format.

Reply:
Well table config is kinda vague, schema is the correct term but then we'd end up with `schema_table_schema` which seems bad too.

Reply:
Ya know what config is fine.